db/import_column_family_job.h

```diff
@@ -22,24 +22,35 @@ namespace ROCKSDB_NAMESPACE {
 struct EnvOptions;
 class SystemClock;
 
+// All file information of an imported CF, mainly used to
+// calculate whether there is overlap between CFs
+struct ColumnFamilyIngestFileInfo {
+  // Smallest internal key in cf
+  InternalKey smallest_internal_key;
+  // Largest internal key in cf
+  InternalKey largest_internal_key;
+  // Num of ingest files
+  int num_files;
```

Comment:
This field seems not used. Maybe we can move this struct to the .cc file under anonymous namespace or inside ImportColumnFamilyJob class.